Nhl.Api/README.md

```
@@ -1898,6 +1968,24 @@ A collection of all the NHL player game shifts for a specific game, including st
 | gameId | [System.Int32](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Int32 'System.Int32') | The game id, Example: 2021020087 |
 | cancellationToken | [System.Threading.CancellationToken](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Threading.CancellationToken 'System.Threading.CancellationToken') | A cancellation token that can be used by other objects or threads to receive notice of cancellation |
 
+<a name='M-Nhl-Api-NhlGameApi-GetPlayoffBracketAsync-System-Int32,System-Threading-CancellationToken-'></a>
+### GetPlayoffBracketAsync(seasonYear,cancellationToken) `method`
+
+##### Summary
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)